webapp/Connector/src/app/view/module/NonIntegratedDataAvailability.js

```diff
@@ -117,36 +112,27 @@ Ext.define('Connector.view.module.NonIntegratedDataAvailability', {
         this.callParent();
     },
 
+    getData : function() {
+        console.warn('need to override getData')
+    },
+
     hasContent : function() {
-        var reports = this.initialConfig.data.model.data.non_integrated_assay_data;
+        console.log('has content');
+
+        // todo delete this unused code
+        var reports = this.getData();
         if (reports) {
```

Comment:
Looks like a "todo" still needs to be handled.

Reply:
yes, done